app/views/WhatYouOwe.scala.html

```diff
@@ -436,9 +436,9 @@
         @if(!whatYouOweChargesList.isChargesListEmpty) {
             @saNoteParagraph
         }
-        @whatYouOweChargesList.balanceDetails.getAbsoluteUnAllocatedCreditAmount.map { unallocatedCreditAmt =>
-            @if(unallocatedCreditAmt > 0) {
-                @unallocatedCreditParagraphContent(unallocatedCreditAmt.toCurrencyString)
+        @whatYouOweChargesList.balanceDetails.availableCredit.map { availableCreditAmt =>
+            @if(availableCreditAmt > 0) {
+                @unallocatedCreditParagraphContent(availableCreditAmt.toCurrencyString)
```

Comment:
<img width="1333" alt="image" src="https://github.com/user-attachments/assets/77deb791-1ef4-439e-9955-5025de44fc05" />

Reply:
as long as we have separation between success response data and viewModel, it should be fine, there still more refactoring need to be done on this page

Reply:
Oh was this just saying to call the whatYouOweChargesList.availableCredit instead of getting it from the balanceDetails? I've made that change and pushed it

Reply:
correct, its a fix and refactoring at the same time